ReportingServicesTools/Functions/CatalogItems/New-RsSubscription.ps1

```diff
@@ -0,0 +1,216 @@
+﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+Function New-RsSubscription {
+    <#
+        .SYNOPSIS
+            This script creates a new reporting subscription.
+        
+        .DESCRIPTION
+            This script creates a new reporting subscription based on the parameters provided. NOTE: A new subscriptionId will be generated.
+                    
+        .PARAMETER Path
+            Specify the path to the destination report.
+        
+        .PARAMETER ReportServerUri
```

Comment:
For consistency sake, I recommend you to add ApiVersion parameter to this cmdlet also.

Reply:
Not sure it yet supports the older APIs for creating subscriptions so am going to leave this for now.